superduper/vector_search/qdrant.py

```diff
@@ -75,16 +76,17 @@ def add(self, items: t.Sequence[VectorItem], cache: bool = False) -> None:
         :param items: List of vectors to add
         :param cache: Cache vectors (not used in Qdrant implementation).
         """
-        points = [
-            models.PointStruct(
+        points = []
+        for item in items:
+            vector = (
+                item.vector.tolist() if hasattr(item.vector, "tolist") else item.vector
```

Comment:
It turns out `item.vector` can be a regular list. For which, `tolist()` doesn't exist.